### PR TITLE
blockdev_stream_install: correct lun_image1

### PR DESCRIPTION
**Pull request by aliang123**
Correct lun_image1 value from 1 to 3
Signed-off-by: Aihua Liang <aliang@redhat.com>
id:2210625

### CODE REVIEW

**Comment by aliang123** [comment]
@zhencliu @XueqiangWei @maxujun, can you help to review it? Thanks.

File: qemu/tests/cfg/blockdev_stream_install.cfg [hunk 1]
```diff
@@ -53,7 +53,7 @@
     # original system image crash when hitting a failure
     image_name_image1 = "images/system_image"
     iscsi_direct:
-        lun_image1 = 1
+        lun_image1 = 3
```
Comment (zhencliu) [inline qemu/tests/cfg/blockdev_stream_install.cfg:56]: hi Aihua,why do we need the change? The default lun is 0 for the image1, then we use lun=1 for another installation could not damage the system image, any other concern?

Comment (aliang123) [reply qemu/tests/cfg/blockdev_stream_install.cfg:56]: sorry for the confusion, @zhencliu, I described the reason in the bug, but not in this patch. Now I update the commit, you can check the reason there.

**Comment by zhencliu** [comment]
Thanks @aliang123 , it looks good to me.

**Review by zhencliu — APPROVED** [review]

**Comment by XueqiangWei** [comment]
LGTM,
Acked-by: xuwei@redhat.com

**Comment by maxujun** [comment]
LGTM，Acked.

**Pull request merged**